### PR TITLE
Issue #146: Add Runserver Command

### LINKED ISSUE #146: Get "runserver" command to work
https://travis-ci.com/alexfinnarn/highlightjs_wysiwyg/builds/81939957#L794

I'm trying to get Behat tests running on PRs via Travis CI working as I've done so on Drupal 7 contrib modules.

The runserver command isn't supported by Backdrop, and I'm not sure why that is or how different that command will need to be for running Backdrop sites. 

I've tried using `php -S` with the built-in server, but that seems to not take into account things like clean URLs and other things I can't remember at the moment...it just didn't work for me.

### PR DESCRIPTION
**Pull request by alexfinnarn**
Fixes #146. I ended up changing the namespace of the command by prepending "backdrop" and I think I got the runserver -> backdrop-runserver part as well.

### CODE REVIEW

File: commands/runserver/backdrop_runserver.drush.inc [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+/**
+ * Implements hook_drush_command().
+ */
+function backdrop_runserver_drush_command() {
+  $items = array();
+
+  $items['backdrop-runserver'] = array(
+    'description' => 'Runs PHP\'s built-in http server for development.',
+    'bootstrap' => DRUSH_BOOTSTRAP_DRUPAL_FULL,
```
Comment (alexfinnarn) [inline commands/runserver/backdrop_runserver.drush.inc:33]: probably need to change Drupal to Backdrop?

File: commands/runserver/backdrop_runserver.drush.inc [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    putenv($key . '=' . $value);
+  }
+
+  // As best I can tell, $extra contains the path to the Drupal router based on
+  // the Drupal major version, 7.x or 8.x. We want to always use the Drupal 7
+  // router.
+  $extra = ' "' . __DIR__ . '/d7-rs-router.php"';
+
+  // Start the server using 'php -S'.
+  $root = drush_get_context('DRUSH_SELECTED_DRUPAL_ROOT');
```
Comment (alexfinnarn) [inline commands/runserver/backdrop_runserver.drush.inc:118]: same there; I don't know if "Drupal" needs to be changed but will try to look

File: commands/runserver/backdrop_runserver.drush.inc [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+      list($name, $value) = explode('=', $value, 2);
+    }
+    $override[$name] = $value;
+  }
+  $env['RUNSERVER_CONF'] = urlencode(serialize($override));
+
+  // NOTE: The original runserver command had a way to log in a user and open a
+  // browser. Those have been removed from the Backdrop version of this command.
+
+  drush_print(dt('HTTP server listening on !addr, port !port (see http://!hostname:!port/!path), serving site !site!user...', array('!addr' => $addr, '!hostname' => $hostname, '!port' => $uri['port'], '!path' => $path, '!site' => drush_get_context('DRUSH_DRUPAL_SITE', 'default'), '!user' => $user_message)));
```
Comment (alexfinnarn) [inline commands/runserver/backdrop_runserver.drush.inc:105]: replace Drupal?

File: commands/runserver/backdrop_runserver.drush.inc (hunk 1 above)
Comment (serundeputy) [reply commands/runserver/backdrop_runserver.drush.inc:33]: yes.

File: commands/runserver/backdrop_runserver.drush.inc (hunk 3 above)
Comment (serundeputy) [reply commands/runserver/backdrop_runserver.drush.inc:105]: probably, but you'll need to test.

**Pull request closed without merging**